openraft/Cargo.toml

```diff
@@ -27,7 +27,7 @@ futures = "0.3"
 maplit = "1.0.2"
 rand = "0.8"
 serde = { version="1", features=["derive", "rc"], optional = true}
-clap = { version = "3.0.7", features = ["derive", "env"] }
+clap = { version = "~3.1", features = ["derive", "env"] }
```

Comment:
This can now be reverted as deprecations are now off by default, see clap-rs/clap#3830 for details.  When you do start working through the deprecations as part of upgrading to 4.0, we've also improved the warnings for derive users.  See clap-rs/clap#3832 for examples.

Reply:
So nice you are!
Thanks!